frontend/scss/components/molecules/teaser-playground.scss

```diff
@@ -0,0 +1,70 @@
+/*
+
+###################################
+### MOLECULE: teaser-playground ###
```

Comment:
"Teaser-Playground" seems super specific. Especially because it could be unified with the "teaser" on the courses overview page.

Reply:
I use teaser-info now, because it is less specific and could be used for showing info about e.g. playground or the course.